drivers/led/led_axp192.c

```diff
@@ -27,9 +27,10 @@ LOG_MODULE_REGISTER(led_axp192, CONFIG_LED_LOG_LEVEL);
 #define FAST_BLINK_DELAY_ON  (((1000 / 4) / 4) * 1)
 #define FAST_BLINK_DELAY_OFF (((1000 / 4) / 4) * 3)
 
-#define CHGLED_CTRL_TYPE_A 0x0
-#define CHGLED_CTRL_TYPE_B 0x1
-#define CHGLED_CTRL_BY_REG 0x2
+#define CHGLED_CTRL_TYPE_A    0x0
+#define CHGLED_CTRL_TYPE_B    0x1
+#define CHGLED_CTRL_BY_REG    0x2
+#define CHGLED_CTRL_BY_CHARGE 0x3
```

Comment:
I think you merged this changes (and those below) into the wrong commit.

Reply:
Nice catch!
I fixed and pushed it.
Thank you.